### PR TITLE
minor man page improvements

### PR DESCRIPTION
**Pull request by garlick**
Here are a few fixes to man page deficiencies that I noticed.

### CODE REVIEW

File: doc/man1/flux-shell.rst [hunk 1]
```diff
@@ -440,6 +440,13 @@ plugins include:
   takes place on all the job's nodes if the scope is local, versus only the
   first node of the job if the scope is global.
 
+.. warning::
```
Comment (chu11) [inline doc/man1/flux-shell.rst:444]: commit message s/beter/better

Comment (chu11) [reply doc/man1/flux-shell.rst:444]: commit message s/drived/drifted/ ?

**Review by chu11 — APPROVED** [review]
LGTM, just those commit message nits i found

File: doc/man1/flux-shell.rst (hunk 1 above)
Comment (garlick) [reply doc/man1/flux-shell.rst:444]: Oof, good catch!

**Comment by garlick** [comment]
Huh, this unit test failed on el8.  I'll run it again but it's an odd one to fail.
```
2024-12-17T19:24:13.6630037Z not ok 261 - 5 file descriptors are open (expected 3-4)
2024-12-17T19:24:13.6630532Z FAIL: test_subprocess.t 261 - 5 file descriptors are open (expected 3-4)
2024-12-17T19:24:13.6631112Z #   Failed test '5 file descriptors are open (expected 3-4)'
2024-12-17T19:24:13.6631414Z #   at test/subprocess.c line 1132.
```

File: doc/man1/flux-shell.rst [hunk 2]
```diff
@@ -319,9 +319,9 @@ plugins include:
   be one of ``term``, ``kvs`` or ``file`` (Default: ``kvs``). If only
   ``output.stdout.type`` is set, then this option applies to both
   ``stdout`` and ``stderr``. If set to ``file``, then ``output.<stream>.path``
-  must also be set for the stream. Most users will not need to set
-  this option directly, as it will be set automatically by options
-  of higher level commands such as :man1:`flux-submit`.
+  must also be set for the stream.
+
+  See also: :option:`flux-submit --output`, :option:`flux-submit --error`.
```
Comment (grondo) [inline doc/man1/flux-shell.rst:324]: Oh, this was smart!

**Review by grondo — APPROVED** [review]
LGTM! Great changes here!

**Comment by garlick** [comment]
Thanks guys - I'll set MWP

**Pull request merged**